util/test/tests/D3D12/D3D12_Multi_Wait_Before_Signal.py

```diff
@@ -0,0 +1,32 @@
+import renderdoc as rd
+import rdtest
+
+class D3D12_Multi_Wait_Before_Signal(rdtest.TestCase):
+    demos_test_name = 'D3D12_Multi_Wait_Before_Signal'
+
+    def check_support(self):
+        # TODO: Enable this if/when rdoc can reorder from the original submission
+        # order, which blocks multiple queues with waits that get signalled by
+        # later submissions to other queues.
+        return False, 'Renderdoc does not yet adequately reorder capture replay'
+
+    def find_all_actions(self, name: str, action_list) -> list:
+        ret_actions = []
+        for action in action_list:
+            if name in self.action_name(action):
+                ret_actions.append(action)
+            ret_actions.extend(self.find_all_actions(name, action.children))
+        return ret_actions
+
+    def check_capture(self):
+        draw_actions = self.find_all_actions('Draw', self.controller.GetRootActions())
```

Comment:
The preferred way to do this is to emit a marker in the demo at the point you want to test (some string to look for, e.g. "Final output") and search for that here.

Alternatively in your case you might just be able to do `self.get_last_action()` assuming it's rendering to the backbuffer.

Reply:
Thanks. Have made the change.
In this case, `get_last_action()` gives me the 'Present(...)', which means `tex = pipe.GetOutputTargets()[0].resourceId` doesn't work, so I've added a `setMarker(...,"Last draw")` and that all works as expected.